src/routes/recipe.ts

```diff
@@ -38,4 +40,24 @@ router.get(
   validate,
   getRecipesLikeByUserId
 );
+router.delete(
+  "/:id",
+  protectedRoute,
+  getParamIdValidator(),
+  validate,
+  (req, res) => {
+    RecipeModel.deleteRecipeById({
+      recipeId: req.recipe.id,
+      userId: req.user.id,
+    })
+      .then((recipeData) => {
+        if (!recipeData) {
+          res.status(500).json({ message: "Did not find that recipe" });
+          return;
+        }
+        res.json(recipeData);
+      })
+      .catch((err) => res.status(400).json(err));
```

Comment:
Let's standardize the error message object
```suggestion
      .catch((err) => res.status(400).send({ error: err});
```